cmd/commands/service.go

```diff
@@ -304,22 +304,25 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 	return command
 }
 
-func ServiceDelete(fcClient *core.Client) *cobra.Command {
+func ServiceDelete(riffClient *core.Client) *cobra.Command {
 
-	deleteServiceOptions := core.DeleteServiceOptions{}
+	options := core.DeleteServiceOptions{}
 
 	command := &cobra.Command{
 		Use:     "delete",
-		Short:   "Delete an existing service",
-		Example: `  ` + env.Cli.Name + ` service delete square --namespace joseph-ns`,
+		Short:   "Delete existing services",
+		Example: `  ` + env.Cli.Name + ` service delete square --namespace joseph-ns
+  ` + env.Cli.Name + ` service delete service-1 service-2`,
 		Args: ArgValidationConjunction(
-			cobra.ExactArgs(serviceDeleteNumberOfArgs),
-			AtPosition(serviceDeleteServiceNameIndex, ValidName()),
+			cobra.MinimumNArgs(serviceDeleteMinNumberOfArgs),
+			StartingAtPosition(serviceDeleteNameStartIndex, ValidName()),
 		),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			fnName := args[serviceDeleteServiceNameIndex]
-			deleteServiceOptions.Name = fnName
-			err := (*fcClient).DeleteService(deleteServiceOptions)
+			names := args[serviceDeleteNameStartIndex:]
+			err := MergeErrors(DeleteResourcesInParallel(names, func(name string) error {
+				options.Name = name
```

Comment:
feedback from @ericbottard: unsafe, instantiate instead a service options struct from the CLI struct there